tests/support/server.tcl

```diff
@@ -735,8 +735,13 @@ proc start_multiple_servers {num options code} {
     uplevel 1 $code
 }
 
-proc restart_server {level wait_ready rotate_logs {reconnect 1} {shutdown sigterm}} {
+proc get_server {level} {
     set srv [lindex $::servers end+$level]
+    return $srv
+}
+
+proc restart_server {level wait_ready rotate_logs {reconnect 1} {shutdown sigterm}} {
+    set srv [get_server $level]
```

Comment:
i don't understand what changed here? or is it just a git diff fuckup?

Reply:
yeah, i'd like that, and i'd also suggest moving this proc to test_helper, but i'm not sure what name or naming convention it should have.